pages/index.tsx

```
@@ -135,19 +135,31 @@ export default function Home() {
-              <button className="border border-white text-white hover:bg-white hover:text-primary-600 font-medium py-3 px-8 rounded-lg transition-colors duration-200">
-                Subscribe
-              </button>
-            </div>
+                         <div className="flex flex-col sm:flex-row gap-4 justify-center">
+               <button 
+                 className="bg-white text-primary-600 hover:bg-primary-50 font-medium py-3 px-8 rounded-lg transition-colors duration-200"
+                 onClick={() => {
+                   // Missing error handling and could be improved
+                   document.getElementById('featured-posts')?.scrollIntoView()
+                 }}
+               >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)